DOCS-2434: Update to uncleared status (#1350)
MultiSafepay/docs: content/faq/risk-and-fraud/accepting-uncleared-credit-card-payments.md

#### An uncleared transaction can only be accepted through your MultiSafepay Control. If no action is taken to accept the uncleared transaction, the authorization will automatically expire after 5 days.
***

> ## Pay after delivery payment methods

Reviewer: ```suggestion
## Post-payment methods
```

> The **Uncleared** status is assigned to a number of [Billing suite](/payment-methods/billing-suite/) payment methods. <br>

Reviewer: ```suggestion
**Uncleared** status applies to a number of post-payment methods in our [Billing Suite ](/payment-methods/billing-suite/). <br>
```

> [Klarna](/payment-methods/billing-suite/klarna/), [Afterpay](/payment-methods/billing-suite/afterpay/), [Betaal per Maand](/payment-methods/billing-suite/betaalpermaand/) and [Pay After Delivery](/payment-methods/billing-suite/pay-after-delivery/) will hold an **Uncleared** status after shipment of an order. The order status will be **Shipped**, while the transaction status will be **Uncleared**. In this case, the transaction is confirmed and the payout is guaranteed. The **Uncleared** status will only change to **Completed** when the payout of a transaction has been received by MultiSafepay and the amount has been added to your MultiSafepay Control balance.

Reviewer: ```suggestion
For [Klarna](/payment-methods/billing-suite/klarna/), [Afterpay](/payment-methods/billing-suite/afterpay/), [Betaal per Maand](/payment-methods/billing-suite/betaalpermaand/) and [Pay After Delivery](/payment-methods/billing-suite/pay-after-delivery/) after you ship the order, the order status is **Shipped** and the transaction status is **Uncleared**. At this point, the transaction is confirmed and settlement is guaranteed. The transaction status then changes to **Completed** when MultiSafepay receives settlement and adds the funds to your MultiSafepay balance.
```

> [Request to Pay](/payment-methods/banks/request-to-pay/), [SOFORT Banking](/payment-methods/banks/sofort-banking/) and [SEPA Direct Debit](/payment-methods/banks/sepa-direct-debit/) will also hold an **Uncleared** status, although the status can be interpreted differently. In this case, the payment of an order has not yet been received by MultiSafepay (paritally or in full e.g. the remaining installments of an in3 payment) and it may take time for the **Uncleared** status to clear. We recommend to __not ship__ items during this period.

Reviewer: ```suggestion
For [Request to Pay](/payment-methods/banks/request-to-pay/), [SOFORT Banking](/payment-methods/banks/sofort-banking/) and [SEPA Direct Debit](/payment-methods/banks/sepa-direct-debit/), **Uncleared** status means MultiSafepay has not yet received settlement (partially or in full, e.g. the remaining instalments of an in3 payment). We recommend **not** shipping orders during this status.
```
